lib/geocoder/lookups/olleh.rb

```diff
@@ -157,24 +162,41 @@ def results(query)
 
     def base_url(query)
       case Olleh.check_query_type(query)
-      when "route_search"
-        "https://openapi.kt.com/maps/etc/RouteSearch?params="
-      when "reverse_geocoding"
-        "https://openapi.kt.com/maps/geocode/GetAddrByGeocode?params="
-      when "convert_coord"
-        "https://openapi.kt.com/maps/etc/ConvertCoord?params="
-      when "addr_step_search"
-        "https://openapi.kt.com/maps/search/AddrStepSearch?params="
-      when "addr_nearest_position_search"
-        "https://openapi.kt.com/maps/search/AddrNearestPosSearch?params="
+      when 'addr_local_search'
+        'https://openapi.kt.com/maps/search/km2_LocalSearch?params='
+      when 'route_search'
+        'https://openapi.kt.com/maps/etc/RouteSearch?params='
+      when 'reverse_geocoding'
+        'https://openapi.kt.com/maps/geocode/GetAddrByGeocode?params='
+      when 'convert_coord'
+        'https://openapi.kt.com/maps/etc/ConvertCoord?params='
+      when 'addr_step_search'
+        'https://openapi.kt.com/maps/search/AddrStepSearch?params='
+      when 'addr_nearest_position_search'
+        'https://openapi.kt.com/maps/search/AddrNearestPosSearch?params='
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.